src/wolff_producer.erl

```diff
@@ -285,8 +278,7 @@ handle_info({do_init, St0}, _) ->
 handle_info(?linger_expire, St) ->
   {noreply, maybe_send_to_kafka(St#{?linger_expire_timer := false})};
 handle_info(?SEND_REQ(_, Batch, _) = Call, #{config := #{max_batch_bytes := Limit}} = St0) ->
-  {Calls, Cnt, Oct} = collect_send_calls([Call], 1, batch_bytes(Batch), Limit),
-  ok = recv_stats(St0, #{cnt => Cnt, oct => Oct}),
+  {Calls, _CollectedBytes} = collect_send_calls([Call], batch_bytes(Batch), Limit),
```

Comment:
Seems like `CollectedBytes` is no longer used anywhere else, so we don't need to keep track of it in `collect_send_calls`?

Reply:
right, this part will be re-written in the next pr.